packages/kolibri-common/strings/enhancedQuizManagementStrings.js

```diff
@@ -85,21 +85,18 @@ export const enhancedQuizManagementStrings = createTranslator('EnhancedQuizManag
     message: '{count, number} {count, plural, one {question selected} other {questions selected}}',
   },
   maxNumberOfQuestions: {
-    message: 'Max { count, number } { count, plural, one { question } other { questions }}',
+    message: 'Maximum number of questions is 50',
```

Comment:
@rtibbles would we prefer to keep this as a dynamic option, rather than changing?

Reply:
Yes, because I still think we should revisit whether 50 is the correct maximum - we've just assumed it, but I also think the user experience of a 50 max is terrible.

Reply:
This parameterization still needs to be reinstated.